tutorials/scripting/scene_unique_nodes.rst

```diff
@@ -40,3 +40,71 @@ name in the path for ``get_node()``. For example:
  .. code-tab:: csharp
 
     GetNode<Button>("%RedButton").Text = "Hello";
+
+Same-scene limitation
+---------------------
+
+A scene unique node can only be retrieved by a node inside the same scene.
+
+If the same scene is instantiated twice, a node inside the first instance
+calling ``get_node("%RedButton")`` will get the button in the first instance,
+not the button in the second instance.
+
+This limitation is perfect for a UI scene like a health bar, where each instance
+of the health bar must not interact with other instances.
+
+When a scene is instantiated inside another scene, the nodes in the outer scene
+can't access scene unique nodes in the inner scene using the scene unique name.
+Likewise, nodes in the inner scene can't access scene unique nodes in the outer
+scene using the scene unique name.
+
+To demonstrate this limitation, consider this example Player scene that
+instances a Sword scene:
+
+.. image:: img/unique_name_scene_instance_example.png
+
+Here are the results of ``get_node()`` calls inside the Player script:
+
+- ``get_node("%Eyes")`` returns the Eyes node.
+- ``get_node("%Hilt")`` returns ``null``.
+
+These are the results of ``get_node()`` calls inside the Blade script:
+
+- ``get_node("%Eyes")`` returns ``null``.
+- ``get_node("%Hilt")`` returns the Hilt node.
+
+If a script has access to a node in another scene, it can call ``get_node()`` on
+that node to get scene unique nodes from that node's scene. This also works in a
+node path, which avoids multiple ``get_node()`` calls. Here are two ways to get
+the Hilt node from the Player script using scene unique nodes:
+
+- ``get_node("Hand/Sword").get_node("%Hilt")`` returns the Hilt node.
+- ``get_node("Hand/Sword/%Hilt")`` also returns the Hilt node.
+
+Scene unique names don't only work at the end of a node path. They can be used
+in the middle to navigate from one node to another. For example, the Sword node
+is marked as a scene unique node in the Player scene, so this is possible:
+
+- ``get_node("%Sword/%Hilt")`` returns the Hilt node.
+
+Alternatives
+------------
+
+Scene unique nodes are a useful tool to navigate a scene. However, there are
+some situations where other techniques may be better.
+
+A :ref:`Group <doc_groups>` allows locating a node from any other node, no
+matter what scene the two nodes are located in. Scene unique nodes are usually
+not suitable to find a Player node from an Enemy node, for example, but a group
+with the Player node in it is a typical solution.
```

Comment:
```suggestion
not suitable to find a **Player** node from an **Enemy** node, for example, but a group
with the **Player** node in it is a typical solution.
```

I'm not sure that a group for just one Player node is actually a typical solution. It would make sense for all enemies to be in a group, but then the example doesn't make much sense for scene unique nodes. Whatever, I guess.

Reply:
I took this example from the PR discussion: https://github.com/godotengine/godot/pull/60298#issuecomment-1100637400. Better example would definitely be nice, but I haven't thought of anything that would be universally understood without some setup. Maybe it's not worth trying to have an example here, and just rely on people clicking through to the groups page without this extra convincing. 😄

Reply:
Removed the example.